library/build.gradle

```diff
@@ -96,7 +96,7 @@ dependencies {
     compile 'com.squareup.moshi:moshi:1.2.0'
     compile 'com.annimon:stream:1.1.1'
     compile 'com.squareup.okhttp3:okhttp:3.4.1'
-    compile 'com.google.protobuf:protobuf-java:3.0.0-beta-3'
+    compile 'com.google.protobuf:protobuf-java:3.5.1'
```

Comment:
If these changes are happening, update the protobuf plugin too.

Reply:
travis passed!!! protobufs 3.5.1 is ok

Reply:
https://github.com/Furtif/POGOProtos/blob/master/.travis.yml#L13